third_party/proton/test/test_profile.py

```diff
@@ -16,7 +16,7 @@ def is_hip():
 
 @pytest.mark.parametrize("context", ["shadow", "python"])
 def test_torch(context, tmp_path: pathlib.Path):
-    temp_file = tmp_path / "test_torch.hatchet"
+    temp_file = tmp_path / f"test_torch_{context}.hatchet"
```

Comment:
@peterbell10 It's not required if `parametrize` isn't used. `tmp_path` is unique per test function but not per input for each function. Feel free to try it out.

Reply:
Without this PR, I'm seeing:
```
/tmp/pytest-of-root/pytest-75/test_torch_shadow_0/test_torch.hatchet
/tmp/pytest-of-root/pytest-75/test_torch_python_0/test_torch.hatchet
```
So the paths are already different?

This was with `pytest 8.3.4` if that matters.

Reply:
You are absolutely right. Maybe my old eyes caused some trouble. :)